plugins/registry/istiov2/cache_test.go

```diff
@@ -0,0 +1,53 @@
+package pilotv2
+
+import (
+	"testing"
+)
+
+var (
+	kubeconfig     string = "/home/lance/.kube/config"
```

Comment:
since there is no mock , write this config file in []byte and write it into file and then use. 
we must let it pass in CI system